en/orm/database-basics.rst

```diff
@@ -261,6 +261,13 @@ work together automatically. By convention, use underscores, lower case, and
 plural forms for your database table names - for example: bakers,
 pastry\_stores, and savory\_cakes.
 
+.. note::
+
+    If your MySQL server is configured with ``skip-character-set-client-handshake``
+    then you MUST use the ``flags`` config to set your charset encoding. For e.g.::
+
+    ``'flags' => [\PDO::MYSQL_ATTR_INIT_COMMAND => 'SET NAMES utf8']``
```

Comment:
```suggestion
        'flags' => [\PDO::MYSQL_ATTR_INIT_COMMAND => 'SET NAMES utf8']
```

You have a mix of inline and block code formatting. This should make it block level only.